MonitoringSystem/DB/MonitoringDB.py

```diff
@@ -20,7 +22,9 @@ class MonitoringDB(ElasticDB):
   """
 
   def __init__(self, name='Monitoring/MonitoringDB', readOnly=False):
-    super(MonitoringDB, self).__init__('MonitoringDB', name, CSGlobals.getSetup().lower())
+    section = getDatabaseSection("Monitoring/MonitoringDB")
+    indexPrefix = gConfig.getValue("%s/IndexPrefix" % section, CSGlobals.getSetup()).lower()
```

Comment:
Maybe mention the option in the module docstring. Similar to: https://github.com/DIRACGrid/DIRAC/pull/4367/files#diff-7ebce825272731fbe4f524930a420d7eR6